ddl/integration_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2015 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ddl_test
+
+import (
+	"fmt"
+	"github.com/juju/errors"
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/domain"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/session"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/store/mockstore"
+	"github.com/pingcap/tidb/terror"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/mock"
+	"github.com/pingcap/tidb/util/testkit"
+	"github.com/pingcap/tidb/util/testleak"
+)
+
+var _ = Suite(&testIntegrationSuite{})
+
+type testIntegrationSuite struct {
+	store kv.Storage
+	dom   *domain.Domain
+	ctx   sessionctx.Context
+}
+
+func (s *testIntegrationSuite) cleanEnv(c *C) {
```

Comment:
we can use `TearDownTest` to avoid calling `cleanEnv` in every tests.

Reply:
I think we'd better clearEnv after every case. It's common that different  cases use the same database name and table name. `test.t` for example. We'd better clean them as soon as possible.

Reply:
`TearDownTest` will be called automatically after every test case, we can rename `cleanEnv` to `TearDownTest` to avoid manually calling this function.

Reply:
ok